src/components/GenreList.jsx

```diff
@@ -10,18 +9,12 @@ const GenreBtn = ({genre, isSelected, onGenreSelection}) => {
     )
 }
 
-const GenreList = ({ genresList }) => {
-    const [selectedGenre, setSelectedGenre] = useState('');
-
-    const onSelect = (genre) => {
-        setSelectedGenre(genre)
-    }
-
+const GenreList = ({ genresList, selectedGenre, onSelectGenre }) => {
```

Comment:
Check also in other places
```suggestion
const GenreList = ({ genresList, selectedGenre, handleSelectGenre }) => {
```